Sources/Dip.swift

```diff
@@ -578,3 +579,68 @@ extension DependencyContainer.Tag: Equatable {
   }
 
 }
+
+
+class DefinitionsContainer {
```

Comment:
should we make this `final`? maybe `private` as well if possible?

Reply:
Final, but not private as its tested.